toolkit/tools/internal/ccachemanager/ccachemanager.go

```diff
@@ -0,0 +1,697 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package ccachemanagerpkg
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/azureblobstorage"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/jsonutils"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/shell"
+)
+
```

Comment:
Could you add a description either here, or spread out through the fuctions, to help people ramping up? Just high level flow stuff.

Reply:
Added.